gprofiler/profilers/java.py

```diff
@@ -186,7 +186,7 @@ def locate_hotspot_error_file(self) -> Optional[str]:
     @functools.lru_cache(maxsize=1)
     def _is_musl(self) -> bool:
         # Is target process musl-based?
-        return any("ld-musl" in m.path for m in self.process.memory_maps())
+        return any("ld-musl" in m.path for m in self.process.memory_maps())  # type: ignore
```

Comment:
Why?

Reply:
because `process.memory_maps` attribute may be unavailable.

Reply:
How didn't it failed the tests until now?
And if it may be unavailable shouldn't we check it?

Reply:
@guybortnikov you are missing `types-psutil`.

If it passes in the CI and doesn't pass on your box, problem is with your box.

Reply:
it was the scandal with darwin, fixed...